_opcodes/op131-bgee.html

```diff
@@ -0,0 +1,37 @@
+---
+n: 131
+name: "State: Positive Chant"
+param1: "Modifier"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Applies the chant effect to the targeted creature(s).<br />
+This opcode should point to <a href="#op22">opcode 22</a> for the effects of "Luck" (or repeat them). It still maintains its own separate <a href="../files/ids/bgee/state.htm#BIT24">STATE</a> though.
```

Comment:
> (or repeat them)
What do you mean by this? If I understand correctly, I suggest this instead:
This opcode has the same effect as <a href="#op22">opcode 22</a>, but maintains its own separate <a href="../files/ids/bgee/state.htm#BIT24">STATE</a>.

And it can be merged back.

Reply:
> I suggest this instead:
> This opcode has the same effect as [opcode 22](#op22), but maintains its own separate [STATE](../files/ids/bgee/state.htm#BIT24). 

Yeah, better, will tweak.